src/search/serializers/paper.py

```
@@ -42,8 +46,8 @@ def get_highlight(self, obj):
     def get_slug(self, hit):
         slug = ''
         try:
-            obj = Paper.objects.get(id=hit['id'])
-            slug = obj.slug
+            paper = Paper.objects.get(id=hit['id'])
+            slug = paper.slug
         except:
             pass
```

can we put a sentry exception here? all errors will be swallowed